SystemContracts/Attributes/HotelAttributes/ItinerarySummary.cs

```diff
@@ -10,7 +10,7 @@ public class ItinerarySummary
 
         public string Name { get; set; }
 
-        public HotelAddress Address { get; set; }
+        public string Address { get; set; }
```

Comment:
what was the need to convert it from object to string?

Reply:
Because there was no need to send a complex address object to the front end as, it would just be used for rendering.

So i am sending just the complete address string